### DISCUSSION TITLE
Can I use KeenDNS certificate for AdGuardHome DoH server

### DISCUSSION THREAD

**Discussion by OPusr** [AdguardTeam/AdGuardHome#3505] [Q&A]
I'm trying to setup an encrypted AdGuardHome DoH server on my Keenetic router using certificate issued for KeenDNS service.
I found certificates in the router rootfs here /tmp/nginx/certs. But it looks to be encrypted.
Did someone found the way to make such setup?

**Comment by frendly** [comment]
@OPusr did you manage to solve the problem?
